src/components/AssignmentTime/types.tsx

```diff
@@ -0,0 +1,32 @@
+import { type Block } from '@ttab/elephant-api/newsdoc'
+import { type LucideIcon } from '@ttab/elephant-ui/icons'
+
+export interface AssignmentValueOption {
```

Comment:
This could probably extend DefaultValueOption
```
interface AssignmentValueOption extends DefaultValueOption {
  info?: string
  slots?: string[]
  median?: string
}
```